questionpy_server/api/models.py

```diff
@@ -32,13 +32,17 @@ class Config:
     tags: Optional[List[str]]
 
 
-class QuestionStateHash(BaseModel):
-    question_state_hash: str
+class OptionalQuestionStateHash(BaseModel):
+    question_state_hash: Optional[str]
     context: Optional[int]
 
 
-class QuestionCreateArguments(BaseModel):
-    form_data: Optional[Dict[str, Any]]
+class QuestionStateHash(OptionalQuestionStateHash):
+    question_state_hash: str
+
+
+class QuestionCreateArguments(OptionalQuestionStateHash):
```

Comment:
Am besten Übernehmen wir die Änderung direkt auch in der [qppe.yaml](https://github.com/questionpy-org/questionpy-server/blob/dev/docs/qppe.yaml#L510-L518).